### PR TITLE
Update primary border tokens

### PR DESCRIPTION
**Pull request by CCAyl**
We had our primary border tokens matching the background tokens, but this is not what we want. this updates:
-border base: grey400
border hover: grey500
border active: grey 800

figma: https://www.figma.com/file/784YociOk102q8PgCz0xAl/Design-Tokens?type=design&node-id=1226-9745&mode=design&t=LkQg8T25WnB6WoJh-4

### CODE REVIEW

**Review by iryanclarke — APPROVED** [review]
Looks good, lets get 'er 🚢

File: src/data/light.json [hunk 1]
```diff
@@ -203,17 +203,17 @@
       },
       "border": {
         "base": {
-          "value": "{colors.grey.700}",
+          "value": "{colors.grey.400}",
           "type": "color",
           "description": "Used for the border colour for primary components such as buttons and Message"
```
Comment (alexandra-lim) [inline src/data/light.json:208]: Should we update the desc as well? Since it won't be true for Buttons anyways

Comment (CCAyl) [reply src/data/light.json:208]: good catch, should i just keep it generic? "Used for the border colour for primary components"?

Comment (alexandra-lim) [reply src/data/light.json]: Looks like it's only being used by CheckableTile for now!

**Review by alexandra-lim — APPROVED** [review]

**Pull request merged**